aion/web/elm/src/Panel/View.elm

```diff
@@ -0,0 +1,36 @@
+module Panel.View exposing (..)
+
+import General.Models exposing (Model)
+import Html exposing (..)
+import Html.Attributes exposing (placeholder, type_, value)
+import Html.Events exposing (onClick, onInput)
+import Msgs exposing (Msg(..))
+import RemoteData exposing (WebData)
+import Room.Models exposing (RoomsData)
+import Select
+
+
+panelView : Model -> Html Msg
+panelView model =
+    div []
+        [ h3 [] [ text "Create new question for certain category:" ]
+        , form []
+            [ p [] [ text "Enter question content below:" ]
+            , input [ placeholder "Zawsze spada na 4 łapy", onInput SetNewQuestionContent ] []
+            , p [] [ text "Enter answer below, separate all posibilities with comma:" ]
+            , input [ placeholder "Kot,kocur,kotek", onInput SetNewAnswerContent ] []
```

Comment:
I wouldn't mix up Polish with English here, maybe change these to whatever in English, just so we stay cohesive